### ISSUE TITLE
After a few days running i'm getting the following error

### ISSUE THREAD

**Issue by TRaSH-** [l3uddz/sonarrAnnounced#10]
http://pastebin.com/xFE925Lr

And the last part continues non-stop.
If i close the program and restart it it starts without any issues.

**Comment by l3uddz** [comment]
is this with the latest version?

**Comment by TRaSH-** [comment]
Yes with the latest release and i've made sure i updated the requirements.txt
actually it's updated before i start the batch file to start sonarAnnounce.

**Comment by l3uddz** [comment]
I'll see if I can solve this. This hasn't happened once to me but I run it on a server. Looks like it's a pydle issue though so will see if they have any suggestions, been very busy lately so haven't had any time to work on this. But will see if can fix this.

**Comment by TRaSH-** [comment]
I notice you're missing tornado in the requirements.txt,
being that [pydle](https://github.com/Shizmob/pydle/blob/master/requirements.txt) needs it.
I've added it to the requirements.txt to see if that solves the issue.
I'll keep you updated.

**Comment by TRaSH-** [comment]
Not sure if the following also is because of the issue i've.

``
2017-02-12 13:00:38,623 - UTILS                - Exception while get_id:
Traceback (most recent call last):
  File "c:\sonarrAnnounced\utils.py", line 27, in get_id
    torrent_id = m[group]
IndexError: list index out of range
``

**Comment by aroundmyroom** [comment]
maybe an issue with the db? try to remove the db and restart and see what is happen?

**Comment by TRaSH-** [comment]
I'll delete it again and give it a restart and see what it does

**Comment by TRaSH-** [comment]
Thought it solved it but after some time same issue :(
